tests/test_remote_workflow.py

```diff
@@ -10,7 +10,7 @@
 import conftest
 
 
-@pytest.fixture(scope="class", autouse=True)
+@pytest.fixture(scope="module", autouse=False)
```

Comment:
@PProfizi shouldn't the servers be shutdown at each test execution for this module ?